### PR TITLE
Add Site Health Info details to bug report template

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change

This PR replaces the lines that request plugin and theme details and instead asks for an export from the Site Health Info page.  We'll get much more complete information from Site Health and as long as folks are running [WP 5.2+](https://make.wordpress.org/core/2019/04/25/site-health-check-in-5-2/) or have the Site Health plugin installed this should be standard (noting that as of today that's [55.7% of installs](https://wordpress.org/about/stats/)).

### Alternate Designs

Keep as-is until WP 5.2+ is a higher percentage of total installs?  Maybe allow for both a Site Health line and additional lines in case Site Health isn't available to self-report plugins/themes?

### Benefits

If we can get Site Health Info details that will provide us with much more info than we normally get and will provide us with valuable information if we're triaging multiple similar issues to try and find potential root causes.

### Possible Drawbacks

A lot more data will get dumped into the description of bug reports.

### Verification Process

Manually verified via the GitHub web UI.

### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

n/a

### Changelog Entry

```
### Changed
- Bug report to request Site Health Info instead of manual plugin/theme reporting
```

### CODE REVIEW

**Comment by dkotter** [comment]
@jeffpaul This looks good to me but I do agree with your point that for those not on WordPress 5.2 or higher, those instructions won't work for them. Not sure if it's worth having extra direction in there that says if you're not on 5.2+, provide this information instead? Or if we're fine just assuming most people submitting reports will be on newer versions of WordPress. Just would hate for someone to not submit a bug report if they just can't figure out what information we're requesting (though acknowledging none of these are required to submit a report, so this may be overthinking things).

**Review by dinhtungdu — COMMENTED** [review]
I agree with @dkotter that we can add a note about the WP version and Site Health plugin, we can remove it when WP 5.2 reaches a higher percentage.

IMO, the information can be added inside `details` tags so it's hidden by default. Issues with (very) long descriptions are hard to follow. For example:

<details>
<summary>Site Health Info</summary>

```
### wp-core ###

version: 5.3.2
site_language: en_US
user_language: en_US
timezone: +00:00
permalink: /%postname%/
https_status: false
user_registration: 0
default_comment_status: open
multisite: false
user_count: 3
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/html
wordpress_size: 37.90 MB (39741350 bytes)
uploads_path: /var/www/html/wp-content/uploads
uploads_size: 48.07 MB (50408558 bytes)
themes_path: /var/www/html/wp-content/themes
themes_size: 3.78 MB (3963640 bytes)
plugins_path: /var/www/html/wp-content/plugins
plugins_size: 957.77 MB (1004298949 bytes)
database_size: 2.56 MB (2686976 bytes)
total_size: 1.03 GB (1101099473 bytes)

### wp-active-theme ###

name: Twenty Twenty (twentytwenty)
version: 1.1
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: automatic-feed-links, custom-background, post-thumbnails, custom-logo, title-tag, html5, align-wide, customize-selective-refresh-widgets, editor-color-palette, editor-font-sizes, widgets, menus, editor-style
theme_path: /var/www/html/wp-content/themes/twentytwenty

### wp-themes-inactive (2) ###

Twenty Nineteen: version: 1.4, author: the WordPress team
Twenty Seventeen: version: 2.2, author: the WordPress team

### wp-plugins-active (2) ###

Classic Editor: version: 1.5, author: WordPress Contributors
ClassifAI: version: 1.4.0, author: 10up

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1690
imagemagick_version: ImageMagick 6.9.10-64 Q16 x86_64 2019-09-16 https://imagemagick.org
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 4 GB
	imagick::RESOURCETYPE_DISK: 9.2233720368548E+18
	imagick::RESOURCETYPE_FILE: 786432
	imagick::RESOURCETYPE_MAP: 4 GB
	imagick::RESOURCETYPE_MEMORY: 2 GB
	imagick::RESOURCETYPE_THREAD: 4
gd_version: 2.2.5
ghostscript_version: not available

### wp-server ###

server_architecture: Linux 4.19.76-linuxkit x86_64
httpd_software: nginx/1.17.8
php_version: 7.3.14 64bit
php_sapi: fpm-fcgi
max_input_variables: 1000
time_limit: 30
memory_limit: 256M
max_input_time: 60
upload_max_size: 150m
php_post_max_size: 150m
curl_version: 7.61.1 OpenSSL/1.1.1c
suhosin: false
imagick_availability: true

### wp-database ###

extension: mysqli
server_version: 5.7.29
client_version: mysqlnd 5.0.12-dev - 20150407 - $Id: 7cc7cc96e675f6d72e5cf0f267f48e167c2abb23 $

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /var/www/html/wp-content
WP_PLUGIN_DIR: /var/www/html/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: true
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: true
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### classifai ###

0: 1.4.0
1: yes
2: tung.du@10up.com
3: 
	Configured: yes
	API URL: https://api.us-south.natural-language-understanding.watson.cloud.ibm.com/instances/cbba6427-bd82-41af-989b-3c862fd23849
	API username: apikey
	Post types: post, page
	Features: {"category":1, "category_threshold":70, "category_taxonomy":"watson-category", "keyword":1, "keyword_threshold":70, "keyword_taxonomy":"watson-keyword", "entity":1, "entity_threshold":70, "entity_taxonomy":"watson-entity", "concept":1, "concept_threshold":70, "concept_taxonomy":"watson-concept"}
	Latest response: {"usage":{"text_units":1, "text_characters":4433, "features":4}, "language":"en", "keywords":10, "entities":5, "concepts":2, "categories":3}
4: 
	Authenticated: yes
	API URL: https://classifai.cognitiveservices.azure.com/
	Caption threshold: 75

```

</details>

**Comment by jeffpaul** [comment]
@dkotter @dinhtungdu I tried to handle for WP>=5.2 and WP<5.2 cases with my latest commit, let me know if you feel that covers your concerns... thanks!

**Review by dinhtungdu — APPROVED** [review]
LGTM 👍

**Review by dkotter — APPROVED** [review]
